NeoML/src/Dnn/Rowwise/MobileNetV2.cpp

```diff
@@ -48,20 +48,23 @@ CRowwiseMobileNetV2::CRowwiseMobileNetV2( IMathEngine& mathEngine ) :
 
 CRowwiseOperationDesc* CRowwiseMobileNetV2::GetDesc()
 {
-	CConstFloatHandle expandFreeTermData = expandFreeTerm == nullptr ? CConstFloatHandle()
+	auto exFreeTermData = ( expandFreeTerm == nullptr ) ? CConstFloatHandle{}
```

Comment:
Reasons to make _any_ changes in these 3 lines?

Reply:
reverted